lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -59,6 +59,29 @@ class SwiftASTManipulatorBase {
     unsigned GetType() override { return Type(); }
   };
 
+  class VariableMetadataPersistent
+      : public SwiftASTManipulatorBase::VariableMetadata {
+  public:
+    VariableMetadataPersistent(
+        lldb::ExpressionVariableSP &persistent_variable_sp)
+        : m_persistent_variable_sp(persistent_variable_sp) {}
+
+    static constexpr unsigned Type() { return 'Pers'; }
```

Comment:
Any idea what this is used for?

Reply:
Looks like the metadata type controls how materialization works. It's only used in `MaterializeVariable`